### PR TITLE
Refactoring emails: part 1

### PR DESCRIPTION
**Pull request by mzylowski**
In the context of #490.

Waiting for review and suggestions.

### CODE REVIEW

File: lib/languages/en.php [hunk 1]
```diff
@@ -1394,8 +1394,7 @@
     'runwatch13' => 'by clicking at',
     'runwatch14' => 'This message is auto-generated - please do not reply',
     'runwatch15' => 'No log entries',
-    'registermail01' => 'Hello',
-    'registermail02' => 'Welcome at',
+    'registermail01' => 'Welcome at',
```
Comment (andrixnet) [inline lib/languages/en.php]: Welcome to

**Comment by andrixnet** [comment]
We should upgrade the HTML email templates as well to take into consideration display on smartphone clients. Currently OC generated emails look very small and almost always require the user to do additional zoom-ins.

File: lib/languages/en.php (hunk 1 above)
Comment (mzylowski) [inline lib/languages/en.php]: Good catch, thanks!

File: Utils/Email/EmailSender.php [hunk 2]
```diff
@@ -44,4 +45,32 @@ public static function adminOnErrorMessage(
...
+        $formattedMessage->setVariable("useruid", $uuid);
+        $formattedMessage->setVariable("server", ApplicationContainer::Instance()->getOcConfig()->getAbsolute_server_URI());
+        $formattedMessage->setVariable("sitename", ApplicationContainer::Instance()->getOcConfig()->getSiteName());
+
+        $formattedMessage->addFooterAndHeader($username);
+
+        $email = new Email();
+        $email->addToAddr($newUserEmailAddress);
+        $email->setReplyToAddr(ApplicationContainer::Instance()->getOcConfig()->getNoreplyEmailAddress());
+        $email->setFromAddr(ApplicationContainer::Instance()->getOcConfig()->getNoreplyEmailAddress());
```
Comment (kojoty) [inline Utils/Email/EmailSender.php]: **in my opinion** this construction if far too complicated:
why to use create instance of ApplicationContainter to get instance of OcConfig?
Just use OcConfig::getNoreplyEmailAddress() method to get this field from config (if there is no such method just implement it)...

Something like ApplicationContainer can be a good solution as a base for Controllers and then provide config object, db object etc... but this is for the further discussion...

**Pull request merged**